### DISCUSSION TITLE
Unmanaged Solution for August Release

### DISCUSSION THREAD

**Discussion by sperry1625** [microsoft/powercat-creator-kit#96] [Q&A] [answered]
The downloads for the August release are different then previous releases.  I do not see an unmanaged solution download for August. Can one be provided, or am I missing something?

Thank you.

-S

**Comment by denise-msft** [comment, marked as answer]
We have decided not to release an unmanaged solution to align with Microsoft's recommended best practices defined in the [solution concepts documentation](https://docs.microsoft.com/en-us/power-platform/alm/solution-concepts-alm#managed-and-unmanaged-solutions). Unmanaged solutions are not intended to be distributed to users who are not implementing changes to the components, and we run into issues for things like updates and deprecations. The intention is to make a more seamless experience. However, we are always open to hearing any reasons why you want to use unmanaged, so if our reasoning does not resonate with you, please let us know!

**Comment by sperry1625** [comment]
I leverage Azure DevOps to manage all solutions and deployments within our environments.

The ALM flows we have established assume a starting point of an unmanaged solution. From there we deploy the solutions to other environments where they are needed. Without that starting point our flows are broken.

Also, even though we do not directly modify the code we do have a requirement to have the source in case this is ever discontinued, we will have what we need to maintain our applications if necessary. Sort of like having the code placed in escrow.

Regards,

-S

**Reply by denise-msft** [reply]
I'm sorry that your DevOps process was broken without an unmanaged solution. Unfortunately, the costs of maintaining an unmanaged version of the components are too high for our dev team to support, which is why we are forced to make that decision for the immediate time being. We can consider this or other ways to support unmanaged solutions when more resources are available for our team. Would a pac cli command to install the latest version of the kit in your devops process be an option for you if we could support that?

For now, is it possible for you to update your DevOps process to read the managed CreatorKitCore solution?

Regarding the source code, everything is available in the GitHub repository and will be available in the history, even in the case the kit is discontinued. You can rebuild the [code components](https://github.com/Microsoft/powercat-code-components) into your own solutions and manage them separately (as unmanged solutions) if that makes more sense.
